components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/Utils.java

```diff
@@ -60,4 +60,10 @@ public static JSONObject createJSONList(int count) {
         jsonBody.put(Constants.LIST, list);
         return jsonBody;
     }
+
+    public static JSONObject createJSONErrorObject(String error) {
```

Comment:
The method should be renamed to `createJsonErrorObject` to match the WSO2 code style.

Reply:
done